cdap-app-fabric/src/test/java/co/cask/cdap/gateway/handlers/OperationsDashboardHttpHandlerTest.java

```diff
@@ -222,46 +228,85 @@ public void testDashboardWithNamespaceFiltering() throws Exception {
 
   }
 
+  List<Long> getExpectedRuntimes(long startTimeInMin, long endTimeInMin, long triggerTimeInMin,
+                                 long actualScheduleStartTime) {
```

Comment:
I think it will be simpler to understand if you call `actualScheduleStartTime` as `testStartTime`

Reply:
updated.